### ISSUE TITLE
中国地图只显示南海诸岛

### ISSUE THREAD

**Issue by penxu** [apache/incubator-echarts#9416]
- [ ] I am using English in this issue.

## General Questions

- [ ] I have read the document and examples and tried to solve it by myself.
- [ ] I have searched for similar issues and that didn't help.
- [ ] I have tried with the latest version of ECharts and still have this problem.

In this issue, I have provided information with:
- [ ] issue type
- [ ] one sentence description in issue details
- [ ] demo if this is bug report

## Issue Type

- [ ] I have a question to ask about *how to use ECharts to ...*
- [ ] I have a bug to report
- [ ] I have a feature to request, e.g.: *I'd like a new feature that ...*
- [ ] I have a feature to enhance, e.g.: *The current feature should be improved in the way that ...*
- [ ] There's something wrong with the documents
- [ ] Others, or I'm not sure which issue types to choose here

## Issue Details

{DESCRIBE_HERE}

### Expected Behavior

{DESCRIBE_HERE}

### Current Behavior

{DESCRIBE_HERE}

### Online Example

{ADD_LINK_OF_DEMO_HERE}

{PASTE_RELATED_CODE_HERE}

## Topics
渲染中国地图时只显示南海诸岛

- [ ] Legend
- [ ] Tooltip
- [ ] Event
- [ ] Performance
- [ ] SVG
- [ ] **###** Map
- [ ] ECharts GL
- [ ] Third-party libraries, e.g.: Vue.js, React
vue.js

## Anything Else We Need to Know

{DESCRIBE_HERE}
配置项如下：
```
  option = { 
    tooltip: {
      trigger: 'item'
    },
    legend: {
      orient: 'vertical',
      left: 'left',
      show: true,
      textStyle: {
        color: '#333'
      }
    },
    visualMap: {
      min: 0,
      max: 2500,
      left: 'left',
      top: 'bottom',
      seriesIndex: [0],
      text: ['高', '低'], // 文本，默认为数值文本
      calculable: true,
      show: false,
      inRange: {
        color: []
      }
    },
    geo: {
      map: 'china',
      label: {
        show: false,
        emphasis: {
          show: false
        }
      },
      roam: true,
      itemStyle: {
        // normal: {
        //   areaColor: '#ccc',
        //   borderColor: '#ddd'
        // }
        emphasis: {
          areaColor: '#ccc'
        }
      }
    },
    series: [
      {
        name: '数值',
        type: 'map',
        mapType: 'china',
        geoIndex: 0,
        label: {
          normal: {
            show: false
          },
          emphasis: {
            show: false,
            textStyle: {
              color: '#fff'
            }
          }
        },
        data: [
          {name: '广东', value: '500'},
          {name: '四川', value: '300'},
          {name: '江苏', value: '1000'}
        ]
      }
}
```
出来的效果只有南海诸岛： 
![2 2fcommon1542332113062 2fpaste1542332048449](https://user-images.githubusercontent.com/23630201/48592408-3a208f00-e983-11e8-8344-9f12c4ffae17.png)
百度了，说没引进china.js
在index.html 引进了
<script src="./src/common/js/china.js" type="text/javascript"></script>
还是不行
## Environment
WIN7
- ECharts version: {DESCRIBE_HERE}
4.0.2
- [ ] It happens only on certain browsers or operating systems. {BROWSER_VERSION_OR_OS_INFORMATION_HERE}

**Comment by Ovilia** [comment]
Yes, the `china.js` is not imported correctly. Please check Chrome Network to see if this file has been loaded correctly.

**Comment by QinZonger** [comment]
文件没有正确导入

**Comment by chfw** [comment]
这里显示的是海南诸岛的轮廓而已。地图没有导入。echarts 在画图的时候，会对china.js区别对待。

```
是 china.js 吗？
哦，是！给画个海南诸岛的轮廓吧！
什么？画脸上啦？
导入了china.js嘛。
我咋知道呢？
js 开发人员工具：查看->js 控制台应该有报错。
还没有看到呢？
js 开发人员工具：查看->js 网络应该有报错。
```

**Comment by youxidonxx** [comment]
使用echarts 5.4.3版本，可能也是没有导入数据的问题，
本人的解决方法如下：
1 先instal 4.x版本，获取node_modules/echarts/map/js和json2个文件出来，然后再安装5.x版本，
并把这两个文件夹复制过来
![image](https://github.com/apache/echarts/assets/1058543/ca4a1ba7-ebb4-4f70-988b-80914b4278cc)
![image](https://github.com/apache/echarts/assets/1058543/15d281a5-57b2-4c60-a34b-07af8f03daa3)
然后在vue界面中引入文件 ，进行注册
![image](https://github.com/apache/echarts/assets/1058543/fd2c848d-1c0f-4bfe-9e5d-6e526321a22f)
![image](https://github.com/apache/echarts/assets/1058543/9e9a8aee-7fa8-4b98-a4d9-ebfdd2e6b274)
2 另一种时使用阿里云的datav的数据源，这种情况下，由于其格式不同，直接拿来用会无法显示地图，需要进行结构转换
![image](https://github.com/apache/echarts/assets/1058543/ef001d6f-0e8c-4306-a959-c9e918635822)
最后效果如下
![image](https://github.com/apache/echarts/assets/1058543/c6cf0cad-9281-4683-92da-ada515465017)

否则一直只有南海的图片
